### DISCUSSION TITLE
"User UID" + key

### DISCUSSION THREAD

**Discussion by kazuser** [SchneiderInfosystems/FB4D#91] [Q&A] [answered]
Wanna use "User UID" as auto-name for a every new key:

![изображение](https://user-images.githubusercontent.com/3335273/116800432-dbcfd280-ab22-11eb-9063-0fe68327be9b.png)

Is it possible? 🤔

**Comment by SchneiderInfosystems** [comment, marked as answer]
Yes, why not? This is a typical application.

**Reply by kazuser** [reply]
Could you please show an example?

**Reply by SchneiderInfosystems** [reply]
I am interested if you have such an example.  I myself currently have no such example for the Firestore. Check the https://github.com/SchneiderInfosystems/FB4D/wiki/Firestore-Chat-Application.
